# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1241885045 -7200
#      Sat May 09 18:04:05 2009 +0200
# Node ID 5fb7e17281e8a6470145818b0903659fffc2dcf4
# Parent  8648058968763a0442ff7bbf4f77838c73a13977
really fix the bug in range

diff --git a/scripts/statistics/base/range.m b/scripts/statistics/base/range.m
--- a/scripts/statistics/base/range.m
+++ b/scripts/statistics/base/range.m
@@ -33,14 +33,14 @@
 ## Author: KH <Kurt.Hornik@wu-wien.ac.at>
 ## Description: Compute range
 
 function y = range (x, dim)
 
   if (nargin == 1)
     y = max (x) - min (x);
   elseif (nargin == 2)
-    y = max (x, dim) - min (x, dim);
+    y = max (x, [], dim) - min (x, [], dim);
   else
     print_usage ();
   endif
 
 endfunction
